src/img/icons.json

```diff
@@ -10,6 +10,7 @@
     "caret-left",
     "caret-right",
     "caret-up-small",
+    "caret-right-small",
```

Comment:
I would move it 2 lines up - right before `caret-right`, so it's grouped together and preserves alphabetical order